clock.rb

```diff
@@ -18,4 +20,10 @@ module Clockwork
   end
 
   every(3.hours, 'scrapping.start')
+
+  Clockwork.manager = DatabaseEvents::Manager.new
+
+  sync_database_events model: Event, every: 15.minutes do |model|
+
```

Comment:
why is this block empty?

Reply:
This field is empty because currently. We do not have action to run :(. I think we should run the task that event dependent on but we do not have this logic yet